packages/packages.json

```diff
@@ -11,31 +11,31 @@
     "protocol/valory/acn/1.1.0": "bafybeifontek6tvaecatoauiule3j3id6xoktpjubvuqi3h2jkzqg7zh7a",
     "protocol/valory/tendermint/0.1.0": "bafybeihcnjhovvyyfbkuw5sjyfx2lfd4soeocfqzxz54g67333m6nk5gxq",
     "skill/valory/abstract_abci/0.1.0": "bafybeiagrbn76jal52v2egtuwelcam3e2huzc6pwjtux2dh5hktxn7em3y",
-    "connection/valory/p2p_libp2p_client/0.1.0": "bafybeihf35zfr35qsvfte4vbi7njvuzfx4httysw7owmlux53gvxh2or54",
```

Comment:
Why has this been changed? It should not have!

Reply:
I don't know, it doesn't make any sense to me on first sight and therefore worries me. I added a separate [Trello ticket](https://trello.com/c/SiaLFpBV/1321-investigate-unexpected-hash-change) for this to the board, I intend to pin point the cause first thing before any additional modifications are introduced in this PR